### PR TITLE
Feature/import components DAO-422 

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- Import button component from ui-components package into web-app
- Setup import alias for ui-components ( `import { abc } from 'ui-components';` )
- Modified ui-components component structure to include `index.tsx` in root component folder

### CODE REVIEW

File: packages/ui-components/tsdx.config.js [hunk 1]
```diff
@@ -1,8 +1,19 @@
 const postcss = require('rollup-plugin-postcss');
+const replace = require('@rollup/plugin-replace');
 
-// TODO: prevent assignment workaround if postcss is being bundled
 module.exports = {
-  rollup(config) {
+  rollup(config, opts) {
+    // suppress prevent assignment warning
```
Comment (sepehr2github) [inline packages/ui-components/tsdx.config.js:6]: Can you explain why you defined it?

File: packages/ui-components/tsdx.config.js [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+    config.plugins = config.plugins.map(plugin =>
```
Comment (sepehr2github) [inline packages/ui-components/tsdx.config.js:7]: Can you explain why you did that?

Comment (sepehr2github) [inline packages/ui-components/tsdx.config.js:7]: Can you explain why you did that?

Comment (Fabricevladimir) [reply packages/ui-components/tsdx.config.js:7]: It's a deprecation warning thrown unconditionally by the rollup/plugin-replace when the ui-component package is being built. You can get more info about it [here](https://github.com/jaredpalmer/tsdx/issues/981#issuecomment-790631832).

**Review by githubdoramon — APPROVED** [review]

**Review by WissenIstNacht — APPROVED** [review]

**Review by sepehr2github — APPROVED** [review]

**Pull request merged**